bowtie/_core.py

```diff
@@ -538,6 +544,74 @@ def supports(self, *dialects: Dialect) -> bool:
         """
         return self.info.dialects.issuperset(dialects)
 
+    async def get_versions(self):
+        from github3 import GitHub  # type: ignore[reportMissingTypeStubs]
+        from github3.exceptions import (  # type: ignore[reportMissingTypeStubs]
+            GitHubError,
+        )
+        from github3.models import (  # type: ignore[reportMissingTypeStubs]
+            GitHubCore,
+        )
+        from github3.structs import (  # type: ignore[reportMissingTypeStubs]
+            GitHubIterator,
+        )
+        from sortedcontainers import (  # type: ignore[reportMissingTypeStubs]
+            SortedSet,
+        )
+
+        url = CONTAINER_PACKAGES_API / self.id / "versions"
+        versions = SortedSet(
+            key=lambda tag: (  # type: ignore[reportUnknownLambdaType]
+                [
+                    -int(part) if part.isdigit() else part
+                    for part in cast(str, tag).split(".")
+                ]
+            ),
```

Comment:
Also @Julian, can you point me somewhere where I can put the above as a `util` itself since this logic is repeated over [here](https://github.com/bowtie-json-schema/bowtie/blob/02c8ea066a2b7931fd97b55ea34c148ad40c5dc8/bowtie/_report.py#L381) as well.

Reply:
I guess here in `_core` considering it's higher up than _report is (and the latter will anyhow import this here).